firebase/example/example.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:firebase_core/firebase_core.dart';
+import 'package:firebase_remote_config_example/home_page.dart';
+import 'package:flutter/material.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry_firebase_remote_config/sentry_firebase_remote_config.dart';
+
+import 'firebase_options.dart';
+
+Future<void> main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  await Firebase.initializeApp(
+    options: DefaultFirebaseOptions.currentPlatform,
+  );
+
+  final remoteConfig = FirebaseRemoteConfig.instance;
+  await remoteConfig.setConfigSettings(RemoteConfigSettings(
+    fetchTimeout: const Duration(minutes: 1),
+    minimumFetchInterval: const Duration(hours: 1),
+  ));
+
+  await SentryFlutter.init(
+    (options) {
+      options.dsn = 'https://example@sentry.io/add-your-dsn-here';
+
+      final sentryFirebaseRemoteConfigIntegration =
+          SentryFirebaseRemoteConfigIntegration(
+        firebaseRemoteConfig: remoteConfig,
+        featureFlagKeys: {'firebase_feature_flag_a', 'firebase_feature_flag_b'},
+        // Don't call `await remoteConfig.activate();` when firebase config is updated. Per default this is true.
+        activateOnConfigUpdated: false,
+      );
+      options.addIntegration(sentryFirebaseRemoteConfigIntegration);
```

Comment:
we gotta take a look if it's possible to track changes automatically without specifying keys

Reply:
We get all the changed keys, so this is not an issue.

Our issue is that we can't check which type the value has (bool, string, number, ...), as the implementation always just falls back to a default value. So for a string value "foobar", we'd just return the default bool value of false, and track this as a feature flag.

https://pub.dev/documentation/firebase_remote_config/latest/firebase_remote_config/RemoteConfigValue/asBool.html

I do not see a way to check for the type dynamically, thats why i introduced the list of keys. 🤔 

But yeah, this way users have to update this list when adding keys on Firebase.

Reply:
Okay I understand the issue. 

then what about this: we always convert the value to a string and then try to evaluate it as a bool. so we get something like `'true'` or `'false'` and convert those to a bool, in all other cases we don't add it to the feature flags.

the only limitation is that actual string values should not be 'true' or 'false' if the user intends to have that value represent something else other than bool - or at least we call that limitation out in the docs.

That way the user doesn't have to pass any keys

Reply:
Ahh, you are completely right, we actually can check if it's bool from the string value, same thing done internally. Thx!